addons/binding/org.openhab.binding.amazondashbutton/README.md

```diff
@@ -0,0 +1,58 @@
+# Amazon Dash Button Binding
+
+The Amazon Dash Button is a very cheap device in order to integrate it in your home automation setup. In openHAB it can be used as a simple switch.
+
+## Prerequisites
+The binding uses pcap4j (https://www.pcap4j.org/) in order to capture ARP requests send by the Amazon Dash Button. The code is inspired by https://github.com/hortinstein/node-dash-button.
+
+
+### Install Native Library
+Let’s start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer. They are native libraries that powers the core functionalities of Pcap4J.
+#### Ubuntu
+```
+apt-get install libpcap-dev
+```
+#### CentOs
+```
+yum install libpcap-devel
+```
+#### Mac
+```
+brew install libpcap
+```
+
+#### Windows
+```
+choco install winpcap
+```
+
+
+Pcap4J needs administrator/root privileges.
+Or, if on Linux, you can run Pcap4J with a non-root user by granting capabilities `CAP_NET_RAW` and `CAP_NET_ADMIN`
+to your java command by the following command: `setcap cap_net_raw,cap_net_admin=eip /path/to/java`
+
+Source: https://www.pcap4j.org/
+
+### Setup Dash Button
+Setting up your Dash button is as simple as following the instructions provided by Amazon **EXCEPT FOR THE LAST STEP**. Just follow the instructions to set it up in their mobile app. When you get to the step where it asks you to pick which product you want to map it to, just quit the setup process.
+
+### Block Internet access for the Dash Button
+Completely deny internet access for the Amazaon Dash Button in your router. You need to find out the Dash button's IP address first of all. This is not explained as this job depends on your environment.
+
+
+## Supported Things
+There is one supported thing:
+* Amazon Dash Button: Thing 
+
+## Discovery
+Background discovery is not supported.
```

Comment:
Hmm, there a lot of traffic floating around, but I can add this feature. I did not test it so far.